### DISCUSSION TITLE
BootstrapFit()

### DISCUSSION THREAD

**Discussion by nsk64** [nlmixrdevelopment/nlmixr#631] [General]
Hi everyone,

Performing a bootstrap n=500 with nonmem often leads to some unsuccessful runs. The terminated runs are not always useful and could be discarded if their number is "small". I do not have in mind an commonly accepted rule.
What about nlmixr? and how to deal with that?

Best regards
Nicolas

**Comment by mattfidler** [comment]
Looking at the code in `nlmixr2extra`, bootstraps that are unsuccessful return `NA` for the parameter estimates, so they are ignored.  I'm unsure what a good number is, but I typically use `10%` myself.
